bedrock/firefox/views.py

```diff
@@ -687,6 +688,8 @@ def get_template_names(self):
 
         if locale == 'en-US' and variant == 'a':
             template = 'firefox/new/trailhead/thanks-a.html'
+        elif ftl_file_is_active('firefox/new/desktop') and switch('new-redesign'):
```

Comment:
Is the switch here just incase something disasterous happens? It's gonna be a little difficult with functional tests as they have no idea about switch state, and not having any form of coverage for these pages is something we should try ot avoid.

Maybe we could enable the switch ahead of pushing, and then only disable is really needed?

Reply:
Tests are updated to work with the switch on or off 🤞